general/ai/cli.go

```diff
@@ -64,67 +60,30 @@ func HowCmd(c *cli.Context) error {
 			}
 		}
 		fmt.Print("\n🤖 Generated command:\n   ")
-		questionBody := QuestionBody{Question: question}
-		llmAnswer, err := askQuestion(questionBody)
+		llmAnswer, err := askQuestion(question)
 		if err != nil {
 			return err
 		}
-		log.Output(coreutils.PrintLink(llmAnswer) + "\n")
-
-		// Ask the user for feedback
-		feedback := FeedbackBody{QuestionBody: questionBody, LlmAnswer: llmAnswer}
-		feedback.getUserFeedback()
-		if err = sendFeedback(feedback); err != nil {
-			return err
-		}
+		log.Output(coreutils.PrintLink(llmAnswer))
 		log.Output("\n" + coreutils.PrintComment("-------------------") + "\n")
 	}
 }
 
-func (fb *FeedbackBody) getUserFeedback() {
-	fb.IsAccurate = coreutils.AskYesNo("Is the provided command accurate?", true)
-	if !fb.IsAccurate {
-		scanner := bufio.NewScanner(os.Stdin)
-		fmt.Print("Please provide the exact command you expected (Example: 'jf rt u ...'): ")
-		for {
-			scanner.Scan()
-			expectedAnswer := strings.TrimSpace(scanner.Text())
-			if expectedAnswer != "" {
-				// If the user entered an expected answer, break and return
-				fb.ExpectedAnswer = expectedAnswer
-				return
-			}
-		}
-	}
-}
-
-func askQuestion(question QuestionBody) (response string, err error) {
-	return sendRequestToCliAiServer(question, questionApi)
-}
-
-func sendFeedback(feedback FeedbackBody) (err error) {
-	_, err = sendRequestToCliAiServer(feedback, feedbackApi)
-	return
-}
-
-func sendRequestToCliAiServer(content interface{}, apiCommand ApiCommand) (response string, err error) {
-	contentBytes, err := json.Marshal(content)
-	if errorutils.CheckError(err) != nil {
-		return
-	}
+func askQuestion(question string) (response string, err error) {
 	client, err := httpclient.ClientBuilder().Build()
 	if errorutils.CheckError(err) != nil {
 		return
 	}
-	req, err := http.NewRequest(http.MethodPost, cliAiApiPath+string(apiCommand), bytes.NewBuffer(contentBytes))
+	req, err := http.NewRequest(http.MethodPost, cliAiAskApiPath, bytes.NewBufferString(question))
 	if errorutils.CheckError(err) != nil {
 		return
 	}
-	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Content-Type", "text/plain")
 	req.Header.Set(apiHeader, "true")
 	log.Debug(fmt.Sprintf("Sending HTTP %s request to: %s", req.Method, req.URL))
 	resp, err := client.GetClient().Do(req)
-	if errorutils.CheckError(err) != nil {
+	if err != nil {
+		err = errorutils.CheckErrorf("CLI-AI server is not available. Please check your network or try again later.")
```

Comment:
In my opinion, it is better to define these errors as constants. This way, you can reuse them in line 100.